internal/provider/provider.go

```diff
@@ -20,13 +21,17 @@ type ImageTestProvider struct {
 	// provider is built and ran locally, and "test" when running acceptance
 	// testing.
 	version string
+
+	// repo exists strictly for testing
+	repo string
```

Comment:
Is there a way to specify a `provider` block in the acceptance test configs so that our tests exercise the same paths that we do in practice? I was wondering about this for my PR as well since I would like to run acceptance tests with different `provider` configs to skip different tests.